unlock_indicator.c

```diff
@@ -261,7 +301,7 @@ xcb_pixmap_t draw_image(uint32_t *resolution) {
     timeinfo = localtime(&rawtime);
 
     if (unlock_indicator &&
-        (unlock_state >= STATE_KEY_PRESSED || auth_state > STATE_AUTH_IDLE || show_clock)) {
```

Comment:
I'll see if I can come up with a compromise (or add a specifier to the clock option or something... I really need to work on a config file)